src/shipit_code_coverage/shipit_code_coverage/report_generators.py

```diff
@@ -33,25 +31,19 @@ def zero_coverage(artifacts, out_dir='code-coverage-reports'):
                     zero_coverage_functions[name].append(f['name'])
                 else:
                     zero_coverage_functions[name] = [f['name']]
-            else:
-                all_functions_uncovered = False
-
-        if all_lines_uncovered or (len(sf['functions']) > 1 and all_functions_uncovered):
-            zero_coverage_files.append(name)
 
-    with open(os.path.join(out_dir, 'zero_coverage_files.json'), 'w') as f:
-        json.dump(zero_coverage_files, f)
-
-    mkdir(os.path.join(out_dir, 'zero_coverage_functions'))
+    mkdir(os.path.join(out_dir, 'zero_coverage_reports'))
 
     zero_coverage_function_counts = []
     for fname, functions in zero_coverage_functions.items():
         zero_coverage_function_counts.append({
             'name': fname,
             'funcs': len(functions),
+            'uncovered': True if len(functions) > 0 else False
```

Comment:
`uncovered` means that the file is completely uncovered, not that there is at least an uncovered function